### DISCUSSION TITLE
Using camelCase

### DISCUSSION THREAD

**Discussion by ahme-dev** [Mokshit06/typewind#26] [Ideas]
First I gotta say this is a really cool library.

Is there any chance you'd be implementing a camelCase option?

**Comment by Mokshit06** [comment]
hey! I'm glad that you like it. I don't think I would be implementing a camelCase option due to the reasons I have mentioned here https://twitter.com/Mokshit06/status/1618235725933334530

**Reply by ahme-dev** [reply]
Good point, but my fingers beg to differ.

**Reply by HananoshikaYomaru** [reply]
@Mokshit06 but would it be able to make it an option to toggle?

**Reply by kavinvalli** [reply]
@HananoshikaYomaru This would introduce a lot of complexities, including recognising arbitrary values syntax which is currently being recognised after the `_`.
